### DISCUSSION TITLE
Cells losing foxus if mouse moves while clicking

### DISCUSSION THREAD

**Discussion by natchovery** [nick-keller/react-datasheet-grid#282] [Q&A]
When you click on a cell to enable edit mode, if you move your mouse even just a little while clicking, it will make the cell looses focus to enter "multi-cell select" mode.

![Animation](https://user-images.githubusercontent.com/128036226/225577729-9644874a-6cad-4044-96b1-f3c54dc99d2c.gif)
(From [example page](https://react-datasheet-grid.netlify.app/docs/getting-started))

As you can see, it makes the Select cell act in a weird way. It automatically closes the dropdown menu.
When you click a bit too long or a bit too fast, it looks like the table is broken.

Is there a way to avoid this behavior ?
Like a way to force the focus on the cell ? Or to disable the "multi-cell select" mode ? Or a "sensitivity" prop ?

Thank you !

(I think this question is somehow linked with #147)

**Comment by nick-keller** [comment]
Good point, I think a `sensitivity` prop (name tbd) in pixel swould be a great idea, can you make a PR?
